packages/fxa-settings/src/components/PageDeleteAccount/en-US.ftl

```diff
@@ -0,0 +1,32 @@
+## Delete account page
+delete-account-header =
+ .title = Delete Account
+
+delete-account-step-1-2 = Step 1 of 2
+delete-account-step-2-2 = Step 2 of 2
+
+delete-account-confirm-title = You've connected your Firefox account to Mozilla products that keep you secure and productive on the web:
+
+mozilla-vpn = Mozilla VPN
+firefox-lockwise = Firefox Lockwise
+firefox-monitor = Firefox Monitor
```

Comment:
nit and maybe we don't need it right now, but having a prefix- for these more generic ids would be a good preventive measure against id clashing in the combined ftl.

Reply:
Yea, I wasn't sure how to handle this because I can see some generic/branding text being used all over.